packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts

```diff
@@ -995,63 +995,44 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
       },
       async detectFaceDistance(context) {
         const {
+          serverSessionInformation,
           isFaceFarEnoughBeforeRecording: faceDistanceCheckBeforeRecording,
         } = context;
-        const { videoEl, videoMediaStream, isMobile } =
```

Comment:
Why is `isMobile` no longer used?

Reply:
Because the value is coming from the server now, previously we would change the values based on mobile or not but now the server provides a distance threshold during before face is detected and a distance threshold during afterwards.